Mutagen.Bethesda.Analyzers.Skyrim/Record/MusicType/NoTracksAnalyzer.cs

```diff
@@ -0,0 +1,32 @@
+using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.MusicType;
+
+public class NoTracksAnalyzer : IIsolatedRecordAnalyzer<IMusicTypeGetter>
+{
+    public static readonly TopicDefinition NoTracks = MutagenTopicBuilder.FromDiscussion(
+            185,
+            "No tracks",
+            Severity.CTD)
+        .WithoutFormatting("MusicType has no tracks");
+
+    IEnumerable<TopicDefinition> IAnalyzer.Topics { get; } = [NoTracks];
+
+    void IIsolatedRecordAnalyzer<IMusicTypeGetter>.AnalyzeRecord(IsolatedRecordAnalyzerParams<IMusicTypeGetter> param)
+    {
+        var record = param.Record;
+
+        if ((record.Tracks is null) || (record.Tracks?.Count == 0))
```

Comment:
Might as well get into some gritty optimization for these analyzers.   Almost everything in the analyzer realm will be accessing readonly record, which are "lazy", which also means they reparse the same data if accessed twice.
https://mutagen-modding.github.io/Mutagen/best-practices/Overlays-Single-Access/#save-to-variable

We can probably swap this to:
```
// One access of `.Tracks` costs one parsing
var tracks = record.Tracks;  

// Accesses of the saved variable don't incur the same cost
if (tracks == null || tracks.Count == 0)
{
   // add topic
}
```

Minor minor stuff, but I think analyzers are the place to "do it right" since they'll probably be running constantly if/when we get "live" mode going